crates/release_plz_core/src/pr.rs

```diff
@@ -2,7 +2,8 @@ use chrono::SecondsFormat;
 
 use crate::PackagesUpdate;
 
-pub const BRANCH_PREFIX: &str = "release-plz/";
+pub const BRANCH_PREFIX: &str = "release-plz-";
```

Comment:
I renamed this from `release-plz_` to `release-plz-`, so that the branch contains only `-` separators.